src/modules/iotjs_module_udp.c

```diff
@@ -409,49 +417,61 @@ void SetMembership(iotjs_jhandler_t* jhandler, uv_membership membership) {
     iface_cstr = iotjs_string_data(&iface);
   }
 
-  int err = uv_udp_set_membership(iotjs_udpwrap_udp_handle(udp_wrap),
-                                  iotjs_string_data(&address), iface_cstr,
-                                  membership);
+  int err = uv_udp_set_membership(iotjs_udpwrap_udp_handle(udp_wrap), address,
+                                  iface_cstr, membership);
 
-  iotjs_jhandler_return_number(jhandler, err);
-
-  iotjs_string_destroy(&address);
   if (!isUndefinedOrNull)
     iotjs_string_destroy(&iface);
+
+  return jerry_create_number(err);
 #else
   IOTJS_ASSERT(!"Not implemented");
 
-  iotjs_jhandler_return_null(jhandler);
+  return jerry_create_null();
 #endif
 }
 
 
 JHANDLER_FUNCTION(AddMembership) {
-  SetMembership(jhandler, UV_JOIN_GROUP);
+  JHANDLER_DECLARE_THIS_PTR(udpwrap, udp_wrap);
```

Comment:
The `AddMemebership` and `DropMembership` methods are almost the same, can't we just pass all jerry arguments to the SetMembership function or create another one which merges the two method's common parts?